lib/styles/trackInputModality.js

```diff
@@ -45,7 +45,7 @@ function setUpEventHandlers() {
       return el.msMatchesSelector;
     }
 
-    logging.error("Couldn't find any matchesSelector method on document.body.");
+    // logging.error("Couldn't find any matchesSelector method on document.body.");
```

Comment:
I temporarily removed logging in the last commit (and pushed this change so that the demo server can be built). The commit will be removed, and I will change this PR from draft to ready to review after I solve the following problem:

After importing `trackInputModality` [here](https://github.com/learningequality/kolibri-design-system/pull/200/files#diff-ab51280bfd8e8d29145c3345af86ef5ef6a52d0b6d676984adbb047a6d43486bR3), where building documentation, I am getting:

```
This dependency was not found
* kolibri.lib.logging in ./lib/styles/trackInputModality.js
```

I tried to fix it by moving `kolibri-tools` to `dependencies` instead of `devDependencies` in *package.json*, without success. Also, I am not sure if it's desirable to bundle `kolibri-tools` with documentation.

Any ideas on how to deal with this.

Reply:
The easiest thing to do is to just delete these logging-related statements. They are a hold-over from when the file was in the Kolibri codebase

For logging in docs code we provide the `consola` library, but that's separate from lib code.